extensions/cornerstone/src/services/CornerstoneCacheService/CornerstoneCacheService.ts

```diff
@@ -11,8 +11,8 @@ const VOLUME_IMAGE_LOADER_SCHEME = 'streaming-wadors';
 const VOLUME_LOADER_SCHEME = 'cornerstoneStreamingImageVolume';
 
 class CornerstoneCacheService {
-  stackImageIds: Map<string, string[]> = new Map();
-  volumeImageIds: Map<string, string[]> = new Map();
+  stackImageIdsMap: Map<string, string[]> = new Map();
```

Comment:
Is it necessary to rename these?

Reply:
it is not a list of imageIds as `stackImageIds` say, it is a map between displaySetInstanceUIDs and their imageIds.